client/dali_grpc_client.py

```diff
@@ -120,44 +127,36 @@ def main():
 
     print("Loading images")
 
-    image_data = load_images(FLAGS.img_dir if FLAGS.img_dir is not None else FLAGS.img)
+    image_data = load_images(FLAGS.img_dir if FLAGS.img_dir is not None else FLAGS.img,
+                             max_images=FLAGS.batch_size * FLAGS.n_iter)
+
     image_data = array_from_list(image_data)
```

Comment:
This converts the image_data to a single np array, and batcher is supposed to work with list via it's doc.
Adjust one way please.